br-ext/package/optee_os_services/Config.in

```diff
@@ -0,0 +1,25 @@
+config BR2_PACKAGE_OPTEE_OS_SERVICES
+	bool "optee_os as package to build service TAs installed in fs"
+	help
+	  Build and install services TAs from the optee_os package
+
+if BR2_PACKAGE_OPTEE_OS_SERVICES
+
+config BR2_PACKAGE_OPTEE_OS_SERVICES_SITE
+	string "OP-TEE OS path"
+	default ""
+	help
+	  The path to the source directory of OP-TEE test
+
+config BR2_PACKAGE_OPTEE_OS_SERVICES_SDK
+	string "OPTEE SDK path"
```

Comment:
Some nits: When it is **not** configuration flags, then please 
s/OPTEE/OP-TEE/

Reply:
done